master/buildbot/reporters/github.py

```diff
@@ -50,7 +50,10 @@ def reconfigService(self, token,
             baseURL = baseURL[:-1]
 
         self._http = yield httpclientservice.HTTPClientService.getService(
-            self.master, baseURL, headers={'Authorization': 'token ' + token})
+            self.master, baseURL, headers={
+                'Authorization': 'token ' + token,
+                'User-Agent': 'buildbot'
```

Comment:
ok changed to "Buildbot". Not that any of this matters because the User-agent isn't being used for anything other than preventing the `403` response from github. In the github example its: `User-Agent: Awesome-Octocat-App`

If we are in the mood for *really* fixing things... https://github.com/Scandy-co/buildbot/blob/master/master/buildbot/reporters/github.py#L132 this has been returning a `403` error, but there's no error handling being done. so looking at the logs, it looks like it worked. going to github... there's no status updates

I made a temporary GitHubStatusPush to diagnose this:

```python
                res = yield self.createStatus(
                    repo_user=repoOwner.encode('utf-8'),
                    repo_name=repoName.encode('utf-8'),
                    sha=sha.encode('utf-8'),
                    state=state.encode('utf-8'),
                    target_url=build['url'].encode('utf-8'),
                    context=context.encode('utf-8'),
                    description=description.encode('utf-8')
                )
                res = yield res.json()
                if self.verbose:
            		  log.msg(
            		        'res: "{res}"'
            		        .format(
            		            res=res))
```